infra/main.bicepparam

```diff
@@ -10,3 +10,5 @@ param principalId = readEnvironmentVariable('AZURE_PRINCIPAL_ID', 'principal_id'
 param useKeyVault = bool(readEnvironmentVariable('USE_KEY_VAULT', 'true'))
 
 param authType = readEnvironmentVariable('AZURE_AUTH_TYPE', 'keys')
+
+param hostingModel = readEnvironmentVariable('AZURE_APP_SERVICE_HOSTING_MODEL', 'container')
```

Comment:
Is this value what is used when someone deploys from the command line? If so, should it instead default to `code`?

Reply:
I've added an extra step in the README. I want the people who dont read the docs to ultimately click "Deploy to Azure" use containers and those that tinker with code to explicitly change it. In the future we could actually supply a Container Registry as part of the build - but trying to keep the scope down.